spec/certificate_request_spec.rb

```diff
@@ -1,22 +1,22 @@
 require 'spec_helper'
 
 describe Acme::CertificateRequest do
-  before(:all) do
-    @test_key = generate_private_key
+  def test_key
+    @test_key ||= generate_private_key
   end
```

Comment:
RSpec seems to do some magic for `before(:all)`, inspecting it, each example still resets the instance variable this way.